internal/controllers/extension_controller.go

```diff
@@ -194,6 +198,8 @@ func (r *ExtensionReconciler) reconcile(ctx context.Context, ext *ocv1alpha1.Ext
 		// originally Reason: ocv1alpha1.ReasonInstallationFailed
 		ext.Status.InstalledBundleResource = ""
 		setInstalledStatusConditionFailed(&ext.Status.Conditions, err.Error(), ext.GetGeneration())
+		setProgressingStatusConditionFalse(&ext.Status.Conditions, "app install failed", ext.GetGeneration())
```

Comment:
This seems like a case where I'd say `Progressing` is still `True` because we encountered a failure but are going to retry.